tox.ini

```diff
@@ -4,6 +4,7 @@
 # E121 - continuation line indentation is not a multiple of four
 # E127 - continuation line over-indented for visual indent
 # E722 - do not use bare 'except'
-ignore = E111,E114,E121,E127, E722
+# W504 - line break after binary operator
```

Comment:
Remove this and fix the indent problem. We are trying to clean up the remaining rules and become flake8 compliant.